keel-sqs/src/test/kotlin/com/netflix/spinnaker/keel/sqs/SqsResourceCheckListenerTests.kt

```diff
@@ -0,0 +1,127 @@
+package com.netflix.spinnaker.keel.sqs
+
+import com.amazonaws.services.sqs.AmazonSQS
+import com.amazonaws.services.sqs.model.DeleteMessageResult
+import com.amazonaws.services.sqs.model.Message
+import com.amazonaws.services.sqs.model.ReceiveMessageResult
+import com.netflix.spinnaker.keel.annealing.ResourceActuator
+import com.netflix.spinnaker.keel.api.ResourceName
+import com.netflix.spinnaker.keel.api.SPINNAKER_API_V1
+import com.netflix.spinnaker.keel.serialization.configuredObjectMapper
+import com.nhaarman.mockitokotlin2.doReturn
+import com.nhaarman.mockitokotlin2.doThrow
+import com.nhaarman.mockitokotlin2.mock
+import com.nhaarman.mockitokotlin2.never
+import com.nhaarman.mockitokotlin2.reset
+import com.nhaarman.mockitokotlin2.stub
+import com.nhaarman.mockitokotlin2.verify
+import dev.minutest.junit.JUnit5Minutests
+import dev.minutest.rootContext
+
+internal class SqsResourceCheckListenerTests : JUnit5Minutests {
```

Comment:
Faster running unit tests that don't deal with any concurrency / coroutine management and just do a single poll on the SQS queue.